src/rules/no-large-snapshots.ts

```diff
@@ -78,28 +103,31 @@ export default {
       },
     ],
   },
-  create(context) {
+  defaultOptions: [{}],
+  create(context, [options]) {
     if (context.getFilename().endsWith('.snap')) {
       return {
         ExpressionStatement(node) {
-          reportOnViolation(context, node);
+          reportOnViolation(context, node, options);
         },
-      };
+      }; // todo: why are we picking out file types here?
```

Comment:
dunno... Open up an issue about it? Seems unnecessary to care about, people should ignore the rule in the files they don't want this rule to run on

Reply:
I've broken this out to #370